pre_commit/commands/try_repo.py

```diff
@@ -23,7 +23,7 @@ def try_repo(args):
         if args.hook:
             hooks = [{'id': args.hook}]
         else:
-            repo_path = Store(tempdir).clone(args.repo, ref)
+            repo_path = Store(tempdir).clone({'repo': args.repo, 'sha': ref})
```

Comment:
Changed clone to take the config dict. Two args which came from the dict was necessarily upped to three, which seemed to be my internal threshold! :)